src/components/material/MaterialHeader.tsx

```diff
@@ -90,7 +90,9 @@ const MaterialHeader: React.FC<MaterialHeaderProps> = ({
           />
         </div>
 
-        {manifestation?.source?.includes("bibliotekskatalog") && (
+        {manifestation?.materialTypes
+          ?.map((i) => i.specific.toLowerCase())
+          .includes("periodikum") && (
```

Comment:
Please move this logic outside of the return value.

It is somewhat complex and should either be moved to an `isPeriocial` boolean variable at the top of this function or create a new `isPeriodical()` helper function.